app/modules/asset_groups/metadata.py

```diff
@@ -483,12 +483,6 @@ def process_request(self):
         self._validate_sightings()
         self.data_processed = AssetGroupMetadata.DataProcessed.sightings
 
-        if not self.bulk_upload and len(self.request['sightings'][0]['encounters']) != 1:
-            raise AssetGroupMetadataError(
-                log,
-                'Incorrect num encounters in form submission, there must be exactly one',
-            )
-
```

Comment:
do we care that we lost the blocking of **zero** encounters here?  i know that _ultimately_ we must have > 1 encounter, but i am not sure if that means it cannot be empty _at this point_ (e.g. 1st encounter gets added later when it is a real sighting).

Reply:
I don't believe that we care. Also the FE will always send one so, no reason that we should